.golangci.example.yml

```diff
@@ -158,6 +158,11 @@ linters-settings:
     # Exclude godoc examples from forbidigo checks.  Default is true.
     exclude_godoc_examples: false
 
+  nlreturn:
```

Comment:
can move this section in this alphabetical place?

Reply:
sure. done.